README.md

```diff
@@ -32,11 +33,11 @@ We test on 16GB GCP GPUs.
 
 ## Quickstart
 
-Here is a simple quickstart to evaluate `hearbaseline` using random
-projections and a tiny subset of the open tasks. More detailed
-instructions are below.
+Here is a simple example to evaluate the `hearbaseline.wav2vec2` model on the
+GTZAN music/speech task. This downloads a preprocessed version of the dataset,
+computes embeddings, and learns a shallow prediction model on the embeddings.
 
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neuralaudio/hear-eval-kit/blob/master/heareval_quickstart.ipynb)
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neuralaudio/hear-eval-kit/blob/master/heareval_example.ipynb)
```

Comment:
Need to add the correct example notebook and link it here properly -- this is a placeholder but the notebook isn't in the repo yet.